HFOSSedu/KitClient
# feat: Terminal asks for student name and adds to prompt.

Closes #31 

This PR will cause the Terminal (`bash`) to prompt the student for their name the first time it is opened.  This name will then become part of the command prompt in the terminal. 

To Verify:
1. remove prior container/image and volume
2. create container as per README (append `:pr-36` to image name)
3. start container as per README
4. connect via vnc or novnc
5. open a terminal
6. should be prompted for name - enter a name
7. prompt should change to include name and pwd.
8. open new terminal - prompt should be same as in 7.
9. stop container
10. restart container 
11. open terminal - prompt should be the same as in 7.
12. stop container as per README
13. delete container as per README
14. create container
15. start container
16. open terminal - prompt should be the same as in 7.
